### ISSUE TITLE
Group and Emergency location dial plan assignment

### ISSUE THREAD

**Issue by CosmoDenger** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#2924]
Please add a way to assign dial plans to both Azure AD groups and Emergency Location Address ID's.  

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 451b44cf-961a-5bf5-e4b3-16b96e008dd7
* Version Independent ID: 54e706e1-aaa2-3a03-b8c6-5e96244b85ac
* Content: [Create and manage dial plans](https://docs.microsoft.com/en-us/microsoftteams/create-and-manage-dial-plans#feedback)
* Content Source: [Teams/create-and-manage-dial-plans.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/create-and-manage-dial-plans.md)
* Service: **msteams**
* GitHub Login: @CarolynRowe
* Microsoft Alias: **crowe**

**Comment by scanum** [comment]
@CosmoDenger  Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by j0rt3g4** [comment]
@CosmoDenger  usually when you add "dial plans" you're assigning a license, the license must be assigned to people not to group.
I understand that a group is composite by people, but there's no direct way to do this 

https://docs.microsoft.com/en-us/azure/active-directory/users-groups-roles/licensing-groups-assign

Related to the Emergency, the same applies you can modify a user at the time:
https://docs.microsoft.com/en-us/skypeforbusiness/what-are-calling-plans-in-office-365/add-change-or-remove-an-emergency-location-for-your-organization


Probably the solution on this case is to create a Powershell Script that gets the group, pipe it to the people inside that group and then run the Cmdlets to assign license (which exists) but sadly changing  Emergency locations has no Cmdlet enabled to do this, and it can be done just using the portal.

**Comment by CosmoDenger** [comment]
Jose,

Can you verify that we are not confusing calling plans with dial plans?  I am asking about dial plans. https://docs.microsoft.com/en-us/microsoftteams/create-and-manage-dial-plans?toc=/skypeforbusiness/sfbotoc/toc.json&bc=/skypeforbusiness/breadcrumb/toc.json

Your first link points out the shortcomings of the current dial plan system exactly.  Dial plans should be setup during the initial rollout and adjusted only occasionally as a new site comes online. If there were dynamic dial plan assignments, we could create a dial plan for each Emergency location and any user assigned to that that location would automatically get the dial plan assigned.

Modifying one user at a time creates unnecessary overhead and the chance to forget to assign a dial plan when adding a new user. As you pointed out, that is the whole purpose of Dynamic assignment.

Basically, we just need a way to use the -LocationID<https://docs.microsoft.com/en-us/powershell/module/skype/set-csonlinevoiceuser?view=skype-ps> parameter to assign a dial plan to it so it would apply for all users assigned to that location.

Thanks,

Cosmo Denger
UA Division of Agriculture
University of Arkansas
Fayetteville, AR 72701
Phone  479.387.3877
agritech.uark.edu<https://agritech.uark.edu/>
Follow us on Twitter: twitter.com/AGRITech_UARK

**Comment by j0rt3g4** [comment]
Hmm I see @scanum  can you please involve the author please?  so he can have the last voice in this

**Comment by scanum** [comment]
@CarolynRowe Hi Carolyn, could you please review this issue. We need your assistance in its resolution. Thanks!

**Comment by CarolynRowe** [comment]
Hmmm...    We have new content about dynamic emergency calling:  https://docs.microsoft.com/en-us/MicrosoftTeams/configure-dynamic-emergency-calling, which is related to Calling Plans--not dial plans.

Let us know if this does not answer your question.

**Comment by CosmoDenger** [comment]
I think this is what we are looking for, although I have not seen any documentation on setting it up in Teams.  Perhaps that will be released shortly and will solve our issue.

[cid:image001.png@01D57CFB.EA8296C0]

Cosmo Denger
UA Division of Agriculture
University of Arkansas
Fayetteville, AR 72701
Phone  479.387.3877
agritech.uark.edu<https://agritech.uark.edu/>
Follow us on Twitter: twitter.com/AGRITech_UARK

**Comment by scanum** [comment]
@CosmoDenger Thank you for submitting feedback. We understand that this issue has been resolved.
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.
